Demo/SelectableText.js

```diff
@@ -1,15 +1,25 @@
 import React from "react";
-import { Text, requireNativeComponent } from "react-native";
+import { Text, requireNativeComponent, Platform } from "react-native";
 
 const RNSelectableText = requireNativeComponent("RNSelectableText");
 
-export const SelectableText = ({ onSelection, ...props }) => {
+export const SelectableText = ({ onSelection, value, children, ...props }) => {
   const onSelectionNative = ({
     nativeEvent: { content, eventType, selectionStart, selectionEnd }
   }) => {
     onSelection &&
       onSelection({ content, eventType, selectionStart, selectionEnd });
   };
 
-  return <RNSelectableText {...props} onSelection={onSelectionNative} />;
+  return Platform.OS === "ios" ? (
+    <RNSelectableText
+      {...props}
+      value={children ? children : value}
+      onSelection={onSelectionNative}
+    />
+  ) : (
+    <RNSelectableText {...props} onSelection={onSelectionNative}>
+      <Text>{children ? children : value}</Text>
```

Comment:
```suggestion
      {children ? children : <Text>{value}</Text>}
```

Probably this would be better if they user is applying some crazy styles on children?